beyond-essence/inventory/modal-dialogs.md

```diff
@@ -10,25 +10,32 @@ Users need to interact with the modal dialog before continuing with further task
 
 ## Use cases
 
-In the Beyond cockpit we provide two types of modal dialogs:
+In the Beyond cockpit we provide two types of modal dialogs.
 
-* Confirmation dialogs. Are used to validate a user's decision.
+* Confirmation dialogs are used to validate a user's decision.
 
 {% image_custom image="/assets/img/pages/essence/modal-dialogs-confirmation.png" width="50" %}
 
-* Input dialogs. Include input fields, checkboxes, or other UI elements the user can interact with.
+* Input dialogs include input fields, checkboxes, or other UI elements the user can interact with.
 
 {% image_custom image="/assets/img/pages/essence/modal-dialogs-inputs.png" width="50" %}
 
 ## Structure
 
 Modal dialogs consist of a container, a close icon, and two buttons.
-The secondary button is always a _Cancel_ button.
-The primary button depends on the specific use case.
+Optionally, they can also contain a title, and further UI elements like checkboxes, or input fields.
+The close icon is always displayed in the upper right corner of the container.
 
-Both buttons are located at the lower right corner of the container whereas the close icon is displayed in the upper right corner.
+The two buttons are located at the lower right corner of the container.
+In confirmation dialogs, these buttons are a primary button, and a danger button.
```

Comment:
You mean in this context the example with the dialog `Page settings. Delete ....` Then we are talking about a default (cancel) and (delete) button.